src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -12,109 +12,119 @@ import ReportActionPropTypes from './ReportActionPropTypes';
 import Clipboard from '../../../libs/Clipboard';
 import {isReportMessageAttachment} from '../../../libs/reportUtils';
 
-/**
- * A list of all the context actions in this menu.
- */
-const CONTEXT_ACTIONS = [
-    // Copy to clipboard
-    {
-        text: 'Copy to Clipboard',
-        icon: ClipboardIcon,
-        successText: 'Copied!',
-        successIcon: Checkmark,
-        shouldShow: true,
-
-        // If return value is true, we switch the `text` and `icon` on
-        // `ReportActionContextMenuItem` with `successText` and `successIcon` which will fallback to
-        // the `text` and `icon`
-        onPress: (action) => {
-            const message = _.last(lodashGet(action, 'message', null));
-            const html = lodashGet(message, 'html', '');
-            const text = lodashGet(message, 'text', '');
-            const isAttachment = _.has(action, 'isAttachment')
-                ? action.isAttachment
-                : isReportMessageAttachment(text);
-            if (!isAttachment) {
-                Clipboard.setString(text);
-            } else {
-                Clipboard.setString(html);
-            }
-        },
-    },
-
-    // Copy chat link
-    {
-        text: 'Copy Link',
-        icon: LinkCopy,
-        shouldShow: false,
-        onPress: () => {},
-    },
-
-    // Mark as Unread
-    {
-        text: 'Mark as Unread',
-        icon: Mail,
-        shouldShow: false,
-        onPress: () => {},
-    },
-
-    // Edit Comment
-    {
-        text: 'Edit Comment',
-        icon: Pencil,
-        shouldShow: false,
-        onPress: () => {},
-    },
-
-    // Delete Comment
-    {
-        text: 'Delete Comment',
-        icon: Trashcan,
-        shouldShow: false,
-        onPress: () => {},
-    },
-];
-
 const propTypes = {
     // The ID of the report this report action is attached to.
     // eslint-disable-next-line react/no-unused-prop-types
     reportID: PropTypes.number.isRequired,
 
     // The report action this context menu is attached to.
-    reportAction: PropTypes.shape(ReportActionPropTypes),
+    reportAction: PropTypes.shape(ReportActionPropTypes).isRequired,
```

Comment:
I think this will cause console errors (I think I remember removing it) because of this: https://github.com/Expensify/Expensify.cash/blob/90030d9f2d2f26d24b9e87377976df45b447a123/src/pages/home/report/ReportActionItem.js#L113

Reply:
Ah, it's safe to pass the `reportAction` in there instead of an empty object, so we could do that _and_ make `reportAction` a required prop.

Reply:
Current code seems to be passing [an action](https://github.com/Expensify/Expensify.cash/blob/main/src/pages/home/report/ReportActionItem.js#L122), so we are good